libs/vm-modules/src/math/tensor/tensor.cpp

```diff
@@ -538,6 +560,9 @@ TensorEstimator &VMTensor::Estimator()
 fixed_point::fp64_t const VMTensor::CONSTRUCTION_PADDED_SIZE_COEF = fixed_point::fp64_t("0.0028");
 fixed_point::fp64_t const VMTensor::CONSTRUCTION_CONST_COEF       = fixed_point::fp64_t("22");
 
+fixed_point::fp64_t const VMTensor::CONSTRUCTION_STRING_SIZE_COEF  = fixed_point::fp64_t("0.12");
```

Comment:
where are these coefficients coming from? if they're based on some benchmarking that you've done locally please make sure that work goes into the ledger-vm-misc repo and maybe we should add a comment here with a link to where the supporting benchmarking can be found justifying these constants?

Reply:
Yes, they are from running a benchmark on my machine. I will do that.